yarn/datadog_checks/yarn/yarn.py

```diff
@@ -231,7 +230,7 @@ def _yarn_app_metrics(self, rm_address, app_tags, addl_tags):
         """
         metrics_json = self._rest_request_to_json(rm_address, YARN_APPS_PATH, addl_tags)
 
-        if metrics_json and metrics_json['apps'] is not None and metrics_json['apps']['app'] is not None:
+        if metrics_json and metrics_json.get('apps') and metrics_json['apps'].get('app') is not None:
```

Comment:
```suggestion
        if metrics_json and metrics_json.get('apps', {}).get('app') is not None:
```

Reply:
Went with checking `metrics_json` first due to json payload being None:
```

tests/test_yarn.py:280: in test_metadata
    check.check(instance)
datadog_checks/yarn/yarn.py:211: in check
    self._yarn_app_metrics(rm_address, app_tags, tags)
datadog_checks/yarn/yarn.py:233: in _yarn_app_metrics
    if metrics_json and metrics_json.get('apps', {}).get('app') is not None:
E   AttributeError: 'NoneType' object has no attribute 'get'
```